integration_test/adminaudit_test.go

```diff
@@ -9,10 +9,12 @@ import (
 	"github.com/supabase-community/gotrue-go/types"
 )
 
-// This test relies on running with an empty audit log.
-// This appears to be fine with the default test settings, but if it becomes an
-// issue locally or in CI, we can add a cleanup step to all tests, or rewrite
-// to stop looking at specific numbers of log entries.
+// This test creates some audit logs, then retrieves them with pagination.
+// However, other tests running in parallel may also create logs, so we only
+// check that _at least_ the logs we create here are returned.
+//
+// In order to test this functionality exactly, we need to run this test against
+// a fresh database with an empty audit log table.
```

Comment:
There are now more audit logs than before, so our other tests create logs. Rather than isolate this test to its own DB, we can just ensure we get _at least_ the logs we expected